resources/styles/components/breadcrumbs/step.less

```diff
@@ -0,0 +1,37 @@
+&-breadcrumbs-step {
+
```

Comment:
This confuses me.  Is it extending whatever the file is included into?  Could we make it just be a class like it was before (`.task-breadcrumbs-step`) ?  I'm guessing you've got a reason for renaming it but I'm missing what it could be :grin: